src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBOptions.java

```diff
@@ -25,18 +25,24 @@ public GenomicsDBOptions(final Path reference) {
     }
 
     public GenomicsDBOptions(final Path reference, GenomicsDBArgumentCollection genomicsdbArgs) {
-        this(reference, genomicsdbArgs, new GenotypeCalculationArgumentCollection(), false);
+        this(reference, genomicsdbArgs, new GenotypeCalculationArgumentCollection());
     }
 
     public GenomicsDBOptions(final Path reference, final GenomicsDBArgumentCollection genomicsdbArgs,
-                             final GenotypeCalculationArgumentCollection genotypeCalcArgs, final boolean forceCallGenotypes) {
+                             final GenotypeCalculationArgumentCollection genotypeCalcArgs) {
         this.reference = reference;
-        this.callGenotypes = genomicsdbArgs.callGenotypes || forceCallGenotypes;
-        this.maxDiploidAltAllelesThatCanBeGenotyped = genomicsdbArgs.maxDiploidAltAllelesThatCanBeGenotyped;
-        this.maxGenotypeCount = genotypeCalcArgs.MAX_GENOTYPE_COUNT;
+        this.callGenotypes = genomicsdbArgs.callGenotypes;
         this.useBCFCodec = genomicsdbArgs.useBCFCodec;
         this.sharedPosixFSOptimizations = genomicsdbArgs.sharedPosixFSOptimizations;
         this.useGcsHdfsConnector = genomicsdbArgs.useGcsHdfsConnector;
+        if (genotypeCalcArgs != null) {
+            this.maxDiploidAltAllelesThatCanBeGenotyped = genotypeCalcArgs.MAX_ALTERNATE_ALLELES;
```

Comment:
This isn't your doing, but it's super confusing that this is in ALL_CAPS because it made me assume it was a constant and not the value that was set.